Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM python:3.8 as base-python
+
+RUN apt-get clean \
+    && apt-get -y update
+
+RUN apt-get -y install nginx \
+    && apt-get -y install python3-dev \
+    && apt-get -y install libpcre3 libpcre3-dev \
+    && apt-get -y install build-essential \
+    && apt-get -y install locales \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN pip3 install 'poetry==1.0.9'
+
+ADD pyproject.toml poetry.lock /
+
+COPY soap_to_rest /soap_to_rest
+RUN chmod +x soap_to_rest/start.sh
+
+FROM base-python as test
+COPY ./tests /tests
```

Comment:
feels weird we're running the tests in the derived docker?  Is this what runs in production?

Reply:
disregard, i see the third image now, the  spacing fooled me

Reply:
The tests are only run when you build the image, the production derived image is what gets published.

Reply:
Might wanna fix that errant newline between the third from and its first command, fooled me and i lookeed over it.